Shell/Skolem.cpp

```diff
@@ -95,6 +96,36 @@ FormulaUnit* Skolem::skolemiseImpl (FormulaUnit* unit, bool appify)
     return unit;
   }
 
+  static bool synthesis = (env.options->questionAnswering() == Options::QuestionAnsweringMode::SYNTHESIS);
```

Comment:
Should this logic be duplicated in NewCNF? I think it has its own Skolem routine. Also potentially definition introduction?

Reply:
Done as discussed offline: we now skolemize the input variables from within SynthesisManager.